ProvisioningTool/src/construct_apdus.cpp

```diff
@@ -591,6 +591,8 @@ int processAttestationIds() {
                 map.add(kTagAttestationIdSerial, std::vector<uint8_t>(idVal.begin(), idVal.end()));
             } else if(0 == key.compare("imei")) {
                 map.add(kTagAttestationIdImei, std::vector<uint8_t>(idVal.begin(), idVal.end()));
+            } else if(0 == key.compare("secondImei")) {
```

Comment:
"not much important" : In case if secondimei is not present in json file then also provision should work smoothly.